editor/src/clj/editor/code/script.clj

```diff
@@ -121,7 +121,10 @@
                                    "string.quoted.other.multiline.lua"
                                    "string.quoted.double.lua"
                                    "string.quoted.single.lua"
-                                   "constant.character.escape.lua"}
+                                   "constant.character.escape.lua"
+                                   "punctuation.definition.comment.lua"
+                                   "comment.block.lua"
+                                   "comment.line.double-dash.lua"}
```

Comment:
Unrelated: I disabled auto-closing parens in comment blocks. Reason: it's common to type English text with single quotes in comments, i.e., words like `don't`.